src/server/blockchain/config.js

```diff
@@ -0,0 +1,7 @@
+require('dotenv').config();
+
+const rpcUrl = process.env.RPC_URL || 'http://localhost:8545';
+
+module.exports = {
```

Comment:
Similar here: Could we stick to `export` syntax for consistency?

Reply:
Same as ^